hri_safety_sense/src/JoystickHandler.h

```diff
@@ -18,34 +18,40 @@
 /**
  * Includes
  */
-#include "ros/ros.h"
+#include "rclcpp/rclcpp.hpp"
 
 #include "hri_c_driver/VehicleMessages.h"
 #include "MsgHandler.h"
 
 namespace hri_safety_sense {
 
-	/**
-	 *
-	 */
-	class JoystickHandler : public MsgHandler {
-	   public:
-		  JoystickHandler(const std::string &frameId);
-		  ~JoystickHandler();
+  /**
+   *
+   */
+  class JoystickHandler : public MsgHandler {
```

Comment:
I know this isn't new in this port, but do we really need the `MsgHandler` abstract class?  If there aren't multiple implementations of the interface, I'd suggest simplifying things and just removing it.

Reply:
I think their intention is to allow the user to write a class like `JoystickHandler` that inherits from `MsgHandler` to fill in the switch cases in `VscProcess.cpp`, which currently have commented hints that they might implement it as `handleGpsMsg()` and `handleFeedbackMsg()`.

The `VehicleMessages.h` in `hri_c_driver` does define types for those, like `GpsMsgType` struct and `VSC_USER_FEEDBACK_KEY_TYPE`, in parallel to `JoystickMsgType` which is used in `JoystickHandler`. They demonstrate usage of GpsMsgType in the C tutorials, but they leave it to the user to implement it in the ROS driver.

Reply:
Added `override` and `final` keywords for interfaces in `JoystickHandler` in 6f083ce